src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
 // write your code here
+const spanElements = document.querySelectorAll('.population');
```

Comment:
it's a good practice to add ref/el suffix to variables with a reference to node element
```suggestion
const spanElementsRef = document.querySelectorAll('.population');
```